op-chain-ops/devkeys/devkeys.go

```diff
@@ -184,6 +186,8 @@ func (role ChainOperatorRole) String() string {
 		return "sequencer-fee-vault-recipient"
 	case SystemConfigOwner:
 		return "system-config-owner"
+	case OperatorFeeVaultRecipientRole:
```

Comment:
Should be able to delete per https://github.com/ethereum-optimism/optimism/pull/12166/files#r1941779240